Tutorials/whatiswot/website/docs/Payloads and Headers.md

```diff
@@ -44,11 +56,11 @@ Configuration Payload:
 }
 ```
 
-Why is JSON important?
+## Why is JSON important?
 
-Because JSON is very relevant in the scope of the Web of Things, payloads are preferred mostly in JSON. Also, many other API and configuration files use JSON.
+Because JSON is very relevant in the scope of the Web of Things, payloads are preferred mostly in JSON. Additionally, many other API and configuration files use JSON as well.
 
-Here, you can see an example of a TD in JSON format. We will go later into the details of how a TD should be structured and understood.
+Here, you can see an example of a TD in JSON format. We will go into the details of how a TD should be structured and understood in a later section of this tutorial.
 
```

Comment:
For the examples below, separate them into a new section. Also adding a human-readable introduction that explains what the data is would be nice